docs/pages/project/contributing/contributing-ui.md

```diff
@@ -96,6 +96,27 @@ make build-ui
 
 > Changes are not recompiled directly, you will have to run to rebuild the UI to see them
 
+
+#### Build Meshery-UI
+
+```
+make build-meshery-ui
+```
+
+> While `make build-ui` builds all user interfaces on your local machine. 
+
+> `make build-meshery-ui` builds only meshery user interface on your local machine.
```

Comment:
@warunicorn19 I don't think we have have to build the UI twice. `make build-ui` builds all of the UI at once so why build the UI again with `make build-meshery-ui`?

Reply:
Makes sense, I will remove it.